pyproject.toml

```diff
@@ -74,6 +74,45 @@ lint.mccabe.max-complexity = 15
 [tool.pytest.ini_options]
 markers = ["all_packages: test install with maximum number of packages"]
 
+[tool.towncrier]
+directory = "changelog.d"
+filename = "CHANGELOG.md"
+start_string = "<!-- towncrier release notes start -->\n"
+underlines = ["", "", ""]
+title_format = "## [{version}](https://github.com/pypa/pipx/tree/{version}) - {project_date}"
+issue_format = "[#{issue}](https://github.com/pypa/pipx/issues/{issue})"
+package = "pipx"
+
+[[tool.towncrier.type]]
+directory = "security"
+name = "Security"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "removed"
+name = "Removed"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "deprecated"
+name = "Deprecated"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "added"
+name = "Added"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "changed"
+name = "Changed"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "fixed"
+name = "Fixed"
+showcontent = true
```

Comment:
Let's make all this inline tables to keep it more compact.

Reply:
I've actually moved to the default fragment types (towncrier has the following default fragment types: feature, bugfix, doc, removal, and misc.), so we don't have to do any configuration at all.
It has the added benefit that this way the towncrier CLI reports which fragment types are available and a short description what they should be used for:
```
-> towncrier create --help

Usage: towncrier create [OPTIONS] FILENAME

  Create a new news fragment.

  Create a new news fragment called FILENAME or pass the full path for a file.
  Towncrier has a few standard types of news fragments, signified by the file
  extension.

  These are:
  * .feature - a new feature
  * .bugfix - a bug fix
  * .doc - a documentation improvement,
  * .removal - a deprecation or removal of public API,
  * .misc - a ticket has been closed, but it is not of interest to users.
```